.github/labeler.yml

```diff
@@ -45,7 +45,7 @@
   - dist/tools/kconfiglib/**/*
   - makefiles/kconfig.mk
   - kconfigs/*
-  - **/Kconfig
+  - ./**/Kconfig
```

Comment:
I see that it is currently failing, but is this going to match all Kconfig files? I'm testing the glob patterns here https://globster.xyz/ is this correct @aabadie ?

Reply:
> is this going to match all Kconfig files?

Probably but I'm not 100% sure. `./` match the current directory then `**/Kconfig` says all Kconfig files in all subdirectories.

Reply:
I checked here now http://pthrasher.github.io/minimatch-test/, which is an online tester for the minimatch library but it does not seem to hit

Reply:
It actually seems like the YAML parser is interpreting the first * as an alias, perhaps we should quote the string?

Reply:
You are probably right. I'll do some tests on my personal fork.